op-conductor/conductor/service.go

```diff
@@ -381,9 +381,7 @@ func (oc *OpConductor) Start(ctx context.Context) error {
 	oc.log.Info("OpConductor started")
 	// queue an action in case sequencer is not in the desired state.
 	oc.prevState = NewState(oc.leader.Load(), oc.healthy.Load(), oc.seqActive.Load())
-	// Immediately queue an action. This is made blocking to ensure that start is not
-	// considered complete until the first action is executed.
-	oc.actionCh <- struct{}{}
+	oc.queueAction()
```

Comment:
revert back to prior behavior, we don't need it to be a blocking behavior here
 especially since it is a buffered channel of size 1